cocotb/handle.py

```diff
@@ -479,6 +479,11 @@ def __le__(self, value):
         Example:
         >>> module.signal <= 2
         """
+        warnings.warn(
+            "Setting values on handles using the ``handle <= value`` syntax is deprecated. "
+            "Instead use the ``handle.value = value`` syntax",
+            DeprecationWarning,
```

Comment:
I wonder if we want to be more gentle about this and use `PendingDeprecationWarning` for one release first.